runners/flink/src/test/java/org/apache/beam/runners/flink/PipelineOptionsTest.java

```diff
@@ -192,8 +192,8 @@ public void parDoBaseClassPipelineOptionsSerializationTest() throws Exception {
             GlobalWindow.INSTANCE,
             PaneInfo.NO_FIRING)));
 
+    testHarness.processWatermark(Long.MAX_VALUE);
```

Comment:
Is this still needed since the check in close() is deleted? Same comments for the other places.

Maybe just revert commit: Add check for output watermark in DoFnOperator...